class/DatabaseConnection.php

```diff
@@ -0,0 +1,340 @@
+<?php
+
+require_once('config/error_messages.php');
+require_once('config/constants.php');
+require_once('error_helper.php');
+
+/**
+* Super Class
+*
+* @author Smruti Ranjan
+*/
```

Comment:
File header should be at start of file
/**
 * @Author : Smruti
 * @purpose : Define database operations
 */